source/install/desktop.rst

```diff
@@ -50,25 +50,52 @@ The Mattermost desktop should open automatically.
 Linux (Beta)
 --------------------------------------------------
 
+Ubuntu and Debian-based systems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Unofficial, community-driven .deb packages are available.
+
 1. Download latest version of the Mattermost desktop app:
 
-   - `x64.tar.gz <https://releases.mattermost.com/desktop/3.6.0/mattermost-desktop-3.6.0-linux-x64.tar.gz>`_
-   - `ia32.tar.gz <https://releases.mattermost.com/desktop/3.6.0/mattermost-desktop-3.6.0-linux-ia32.tar.gz>`_
+    64-bit systems
+     `mattermost-desktop-3.6.0-linux-amd64.deb <https://releases.mattermost.com/desktop/3.6.0/mattermost-desktop-3.6.0-linux-amd64.deb>`_
+    32-bit systems
+     `mattermost-desktop-3.6.0-linux-i386.deb <https://releases.mattermost.com/desktop/3.6.0/mattermost-desktop-3.6.0-linux-i386.deb>`_
+
+2. At the command line, execute one of the following commands depending on the package that you downloaded:
 
-2. Extract the archive, then execute ``Mattermost`` which is located inside the extracted directory.
+    64-bit systems
+      ``sudo dpkg -i mattermost-desktop-3.6.0-linux-amd64.deb``
+    32-bit systems
+      ``sudo dpkg -i mattermost-desktop-3.6.0-linux-i386.deb``
 
-3. If you need a Desktop Entry for the application to integrate into a desktop environment, please refer to https://wiki.archlinux.org/index.php/Desktop_entries
+3. To run Mattermost, open **Dash** (located at top left corner) and input ``mattermost``, then click the Mattermost icon.
 
-To download the unofficial, community-driven .deb packages for the desktop application:
+Generic Linux package
```

Comment:
Can we move the generic Linux package first, since the Debian packages are unofficial?